### ISSUE TITLE
A detailed guide for avoiding conflicts with swupd update/repair

### ISSUE THREAD

**Issue by insilications** [clearlinux/clear-linux-documentation#913]
For example, while it is pretty explicit for a Linux newbie that one can avoid a potential conflict with swupd down the road by using /usr/local paths, I didn't know that swupd leaves /opt/* alone as well. In general, a more detailed guide for those who want to enjoy the stateless design while maintaining their personalized installation intact.

**Comment by insilications** [comment]
Thanks for correcting me and moving this to [clear-linux-documentation](https://github.com/clearlinux/clear-linux-documentation). 

@lebensterben @mrkz perhaps we should also specify that */opt* is also a user managed area in the stateless guide? 

Also, wouldn't it be important to update the output for "swupd repair --help"? See:
`Options:`
` -Y, --picky             Also remove files which should not exist`
`-w, --picky-whitelist=[RE] Directories that match the regex get skipped. Example: /var|/etc/machine-id Default: /usr/lib/modules|/usr/lib/kernel|/usr/local|/usr/src`

The default list shouldn't also list /etc/, /home, /var and /opt?

**Comment by lebensterben** [comment]
>Thanks for correcting me and moving this to [clear-linux-documentation](https://github.com/clearlinux/clear-linux-documentation).  ...

@insilications I think by convention most distros assume /opt is  managed only by the user. But I agree it makes it even clearer if the documentation explicitly mentioned this.

As for the suggestion on the help message, you may open an issue in clearlinux/swupd-client repository.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by intelkevinputnam** [comment]
@lebensterben and @insilications thanks for continuing to contribute to the conversation. Since there is a PR in progress, please take a look and make any additional comments there (#912).

**Comment by mrkz** [comment]
@insilications Hey, so the PR I created for the stateless web page guide is merged and should be available soon-ish when the content is updated in the web page. Additionally, I created https://github.com/clearlinux/swupd-client/issues/1205 as a follow-up on your comment above.

I'm closing this issue as one of the tho items is addressed already and the missing one is being handled on the swupd-client repository.
